osbs/conf.py

```diff
@@ -426,3 +427,30 @@ def get_oauth2_token(self):
     def get_reactor_config_secret(self):
         return self._get_value("reactor_config_secret", self.conf_section,
                                "reactor_config_secret")
+
+    def get_client_config_secret(self):
+        return self._get_value("client_config_secret", self.conf_section,
+                               "client_config_secret")
+
+    def get_token_secrets(self):
+        value = self._get_value("token_secrets", self.conf_section,
+                                "token_secrets")
+        token_dict = {}
+        will_raise = False
+        if value:
+            for pairs in value.split():
+                pair = pairs.split(':', 1)
+
+                if len(pair) == 2:
+                    if pair[1] in "/":
```

Comment:
That's pretty non-obvious I think. I'd prefer to make this explicit, e.g. `if pair[1] in ['', '/']:`.